Packs/AzureSentinel/Scripts/MicrosoftSentinelSetOwnerButton/MicrosoftSentinelSetOwnerButton.yml

```
@@ -0,0 +1,30 @@
+args:
+- name: owner_email
+  required: true
+  description: The owner email to set as user principal name.
+commonfields:
+  id: MicrosoftSentinelSetOwnerButton
```

```suggestion
  id: MicrosoftSentinelSetOwner
```
and change names of files not to be specific to button. THis script can also be called not by the button

---

Done